bin/rule-generator.js

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const figlet = require('figlet');
+const inquirer = require('inquirer');
+const cmd = require('node-command-line');
+const axeFile = path.join(__dirname, '..', 'axe.js');
+const filehound = require('filehound');
+
+const DIRECTORIES = {
+	RULES: path.join(__dirname, '..', '/lib/rules'),
```

Comment:
Should replace the path separators here and let the `path` module handle it for you. This helps support other platforms (Windows).

For example:

```js
{
  RULES: path.join(__dirname, '..', 'lib', 'rules'),
  // ...
}
```